subsys/bluetooth/controller/ll_sw/ull_chan.c

```diff
@@ -23,6 +23,7 @@
 #include "lll/lll_adv_types.h"
 #include "lll_adv.h"
 #include "lll/lll_adv_pdu.h"
+#include "lll/lll_df_types.h"
```

Comment:
This is a modification I had to make to a non-UT source file in order to get it to build (it was missing a struct declaration otherwise)